doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -84,37 +175,265 @@ writeTextFile {
   executable = true;
   destination = "/bin/my-file";
 }
-# Writes contents of file to /nix/store/<store path>
+```
+:::
+
+::: {.note}
+The commands `writeText`, `writeTextDir`, `writeScript`, `writeScriptBin`, `writeShellScript`, and `writeShellScriptBin` documented below are convenience functions that wrap `writeTextFile`.
+:::
+
+### `writeText` {#trivial-builder-writeText}
+
+Writes a text file to the store
+
+`writeText` takes two arguments: `name` and `text`, each of which should be
+a string.
+
+`name`
+
+: the name used in the Nix store path.
+
+`text`
+
+: will be the contents of the file.
+
+The store path will include the the name, and it will be a file. Any path separators and shell-reserved elements in the name are escaped to create the store path identifier.
```

Comment:
What is meant by "it will be a file" in "The store path will include the name, and it will be a file."?

Edit, reading further examples like this down below, I guess I understand that this is trying to communicate: the `outPath` of the derivation is a file. (vs. the `outPath` is a directory).

Not sure if there's a clearer way to say this, because for some reason, it was confusing to me initially. Just an FYI; not important.